packages/binding/src/types/issues.ts

```diff
@@ -0,0 +1,82 @@
+import { BaseUI5XMLViewIssue } from "@ui5-language-assistant/xml-views-validation";
+import { Range } from "vscode-languageserver-types";
+import { BINDING_ISSUE_TYPE } from "../constant";
+
+export type BindingIssue =
+  | MissingKey
+  | MissingColon
+  | TooManyColons
+  | MissingValue
+  | MissingComma
+  | TooManyCommas
+  | Syntax
+  | TrailingComma
+  | UnknownChar
+  | UnknownPropertyBindingInfo
+  | MissMatchValue
+  | NotAllowedProperty
+  | DuplicateProperty
+  | RequiredDependency
+  | RecursiveProperty
+  | MissingLeftCurly;
+
+interface BaseUI5XMLViewBindingIssue extends BaseUI5XMLViewIssue {
```

Comment:
Since we have a different handling for external validation issues, do we really need to extend the base type? This forces us to convert back and forth between ranges and offsets which adds unnecessary coding.